### PR TITLE
Logging improvements and documentation update

### PR DESCRIPTION
**Pull request by mockitoguy**
Prepping for 1.0! :)

### CODE REVIEW

File: docs/how-shipkit-works.md [hunk 1]
```diff
@@ -4,111 +4,22 @@ Want to find out how Shipkit works?
...
-#### Intro
+### Details
 
-Shipkit needs to be able to automatically use the right version when building and publishing.
-This way, you don't need to deal with changing/updating the version for every release.
-Every release will produce n+1 version.
+How do we:
+- [bump versions](/docs/features/version-bumps.md)
+- [generate release notes](/docs/features/automated-release-notes.md)
+- [publish binaries](/docs/gradle-plugins/publishing-binaries.md)
```
Comment (epeee) [inline docs/how-shipkit-works.md]: This link seems to be broken. It should rather be `docs/features/publishing-binaries.md`, isn't it?

File: docs/how-shipkit-works.md [hunk 2]
```diff
@@ -4,111 +4,22 @@ Want to find out how Shipkit works?
...
+- [publish binaries](/docs/gradle-plugins/publishing-binaries.md)
```
Comment (epeee) [inline docs/how-shipkit-works.md]: duplicate "publish binaries" link

File: README.md [hunk 3]
```diff
@@ -69,6 +69,8 @@ You have code that wants to be delivered to your customers, Shipkit has tools to
 - keeping secure tokens safe - masks sensitive values from logging and error messages from underlying tools like git
 - and all that in a sweet little package, fully integrated, neatly automated, easy to roll out
 
+For details see "[How Shipkit Works?](/docs/how-shipkit-works.gradle)"
```
Comment (epeee) [inline README.md]: Should be "/docs/how-shipkit-works.md", isn't it?

**Comment by mockitoguy** [comment]
Great feedback, just pushed the fixes. Thank you!

**Comment by mockitoguy** [comment]
I will merge this ticket if that's ok. The changes are mainly documentation. I have addressed @wwilk's feedback.

**Pull request merged**